package.json

```diff
@@ -16,25 +16,25 @@
         "url": "https://github.com/cloudconvert/cloudconvert-node/issues"
     },
     "dependencies": {
-        "axios": "^0.21.1",
+        "axios": "^0.26.1",
         "esm": "^3.2.25",
```

Comment:
This should be a dev dep, no?

Reply:
Yes, should be dev dep

Reply:
Alright, I'll update it in a few hours when I have time, and then it's ready to be merged